api/graphql/mutations/group.js

```diff
@@ -115,7 +116,20 @@ export async function removeModerator (userId, personId, groupId, isRemoveFromGr
 
 export async function updateGroup (userId, groupId, changes) {
   const group = await getModeratedGroup(userId, groupId)
-  return group.update(convertGraphqlData(changes))
+  const convertedChanges = convertGraphqlData(changes)
+  /* 
+    - check if customView is in changes
+    - check if customView exists
+    - either update/create customView
+  */
+ 
+  if (group && convertedChanges.customView) {
```

Comment:
I feel like mostly we handle this type of extra work in the Group.update function itself instead of in the mutation, is there a reason you put this here?

Reply:
yep, this was just me following the chain of where things happening and stopping too early as I went along the chain! Sorry! Moving this